src/config.rs

```diff
@@ -261,9 +270,22 @@ impl Config {
 
         config.loaded_from.push(location.to_path_buf());
 
+        config.validate()?;
+
         Ok(config)
     }
 
+    /// Validate the config file.
+    pub fn validate(&self) -> miette::Result<()> {
+        if let Some(target_env_dir) = self.target_environments_directory.clone() {
+            if !target_env_dir.exists() {
```

Comment:
This check doesnt actually check if the `target_env_dir` is absolute, so `./some_dir` would also work.

Reply:
Now also checking if it is absolute.